Solutions/MCBSTM32F400/DeviceCode/Crypto/dotNetMF.proj

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <AssemblyName>Crypto_config_MCBSTM32F400</AssemblyName>
+    <Size>
+    </Size>
+    <ProjectGuid>{8E104C8A-D48B-42F2-8DB6-E1AF5937C34F}</ProjectGuid>
+    <Description>PKCS11 Crypto configuration for MCBSTM32F400</Description>
+    <Level>HAL</Level>
+    <LibraryFile>Crypto_config_MCBSTM32F400.$(LIB_EXT)</LibraryFile>
+    <ProjectPath>$(SPOCLIENT)\Solutions\MCBSTM32F400\DeviceCode\Crypto\dotNetMF.proj</ProjectPath>
+    <ManifestFile>Crypto_config_MCBSTM32F400.$(LIB_EXT).manifest</ManifestFile>
+    <Groups>Solutions\MCBSTM32F400</Groups>

```

Comment:
This was cloned from the i.MXS solution which doesn't have one, but if you've got details on that let me know and I can incorporate it.